app/controllers/get_involved_controller.rb

```diff
@@ -0,0 +1,94 @@
+class GetInvolvedController < ApplicationController
+
+  attr_accessor :content_item
+  
+  def show
+    load_content_item
+    load_get_involved_data
+    @do_not_show_breadcrumbs = true
+    render template: "content_items/get_involved"
+  end
+
+  def load_content_item
+    content_item = Services.content_store.content_item("/government/get-involved")
+
+    if Services.feature_toggler.use_recommended_related_links?(content_item["links"], request.headers)
+      content_item["links"]["ordered_related_items"] = content_item["links"].fetch("suggested_ordered_related_items", [])
+    end
```

Comment:
This feels like it needs a test.

Reply:
Resolving as this code is no longer present.